feat(unity): Added Unity to screenshots (#4968)
getsentry/sentry-docs: src/includes/enriching-events/attach-screenshots/unity.mdx

> The **Attach Screenshot** options can be found on the `Enrichment` tab in the Editor Configuration Window.

Reviewer: ```suggestion
The "Attach Screenshot" options are on the "Enrichment" tab of the editor configuration window.
```

![Screenshots Thumbnail Example](unity-screenshots.png)

<Note>

> Note that capturing a screenshot in the Unity SDK requires running on the main thread.

Reviewer: Making a wording change here to try to clarify, but if I've done so incorrectly, let me know.

```suggestion
Capturing a screenshot in Unity requires running the SDK on the main thread.
```

> Note that capturing a screenshot in the Unity SDK requires running on the main thread.

Reviewer 2: This is important to be written in the common docs as well (src/platforms/common/enriching-events/screenshots.mdx)

</Note>

> ### Max Width and Max Height

Reviewer 2: Please amend the specs with this 2 configs https://develop.sentry.dev/sdk/features/#screenshots

Reviewer 2: I'd also add this new 2 options in here below https://docs.sentry.io/platforms/android/configuration/options/#attach-screenshot

> The default value of 0 means that screenshots get captured in the game's current [Screen](https://docs.unity3d.com/ScriptReference/Screen.html) size. With max values greater than 0 but less than the screen size, the SDK will try to scale down the captured screenshot while preserving the apect ratio. This allows for a smaller file being sent.

Reviewer: I've made a change to the last sentence that I'm hoping doesn't change the meaning too much. If it does, lemme know and I'll try to edit a different way.
```suggestion
The default value of 0 means that screenshots are captured in the game's current [screen](https://docs.unity3d.com/ScriptReference/Screen.html) size. If max values are greater than 0 but less than the screen size, the SDK will try to scale down the captured screenshot while preserving the aspect ratio. This allows for a smaller file to be generated.
```
